cardano-node/src/Cardano/Node/Tracing/Compat.hs

```diff
@@ -0,0 +1,22 @@
+{-# LANGUAGE LambdaCase #-}
+
+module Cardano.Node.Tracing.Compat
+  ( toDetailLevel
+  , fromDetailLevel
+  ) where
+
+import qualified Cardano.BM.Data.Tracer as IOMF
+import qualified Cardano.Logging.Types as TD
+
+toDetailLevel :: IOMF.TracingVerbosity -> TD.DetailLevel
```

Comment:
This can also be moved to `Cardano.Node.Tracing`

Reply:
Same as above. It is obviously possible and more a matter of taste.

Reply:
Alright then lets move it to `Cardano.Node.Tracing.API` for the time being. It really doesn't justify a new module at the moment.

Reply:
This entire module will be removed with old-tracing, and it is cleaner to keep it seperate.

Reply:
Ok fair enough :+1: